feat: update stencil version and improve logging (#72)
siemens/ix-icons: .changeset/light-schools-buy.md

---
'@siemens/ix-icons': minor
---

> - Improve logging include html element inside logging to identify which component creates the logging

Reviewer: ```suggestion
- Improve logging by including HTML element in output to identify the source component.
```
